### DISCUSSION TITLE
'Billing Code Type Version' - format for this field ?

### DISCUSSION THREAD

**Discussion by ABPhase2** [CMSgov/price-transparency-guide#575] [General]
Hi all,

Seeking clarification on the format for the field 'Billing Code Type Version'. What is the format to be used for this field are both YYYY and YYMM acceptable  or is it required that the field is in the YYMM format ? 
Have there been any insights on this shared by CMS ? 

Billing Code Type version field e.g. YYYY : 2022 or YYMM 2208 (Aug 2022).

Thank you.

**Comment by dgolden** [comment]
I do not know if CMS has shared guidance on this. However I did a quick check on an Arkansas BCBS and only see YYYY (plus MS-DRG). See the attached JPEG for my query results and MRF identification.

![billing_code_type_version](https://user-images.githubusercontent.com/6385204/183747917-54f618d5-c430-43ea-b93e-b753e4a47d2a.jpg)

**Comment by shaselton-usds** [comment]
The documentation was assuming (which will need to be updated to be explicit) the format to be YYYY, as are in the examples and in the main documentation example.

Small background discussion: https://github.com/CMSgov/price-transparency-guide/discussions/203#discussioncomment-2702738
